holdem/utils.py

```diff
@@ -29,6 +30,26 @@ class action_table:
   FOLD = 3
   NA = 0
 
+class community_table:
```

Comment:
I'm fine with this, you can make this inherit `Enum`:
```python
class community_table(Enum):
    # declarations here
```
([docs for enum](https://docs.python.org/3/library/enum.html))

Reply:
Got inspiration for it from another fork of the repo. Not really sure which one it was.

Reply:
Made all of these classes inherit from IntEnum